test/test_out_dtype_op.py

```diff
@@ -121,14 +120,15 @@ def f(x, y):
         self.assertTrue(torch.allclose(numerical_res, gm(*inp)))
 
     def test_out_dtype_non_functional(self):
-        def f(x, y):
-            return out_dtype(
-                torch.ops.aten.add_.Tensor, torch.int32, x, y
-            )
+        class M(torch.nn.Module):
```

Comment:
function no longer supported?

Reply:
Not since https://github.com/pytorch/pytorch/pull/117528